_templates/exchanges.html

```diff
@@ -55,7 +55,8 @@ <h3 id="australia"><img src="/img/flags/AU.png" alt="Australian flag">Australia<
         <a href="https://www.coinloft.com.au/">CoinLoft</a><br>
         <a href="https://www.cointree.com.au/">CoinTree</a><br>
         <a href="https://www.hardblock.net/">HardBlock</a><br>
-        <a href="https://www.independentreserve.com/">Independent Reserve</a>
+        <a href="https://www.independentreserve.com/">Independent Reserve</a><br>
+	<a href="https://www.coinspot.com.au/">CoinSpot</a>
```

Comment:
@gg2001 

Each country's exchanges are listed alphabetically. I just noticed that this isn't the case here on this PR. So just to confirm you should:

1. Remove the `<br>` from Independent Reserve and leave it as the last entry.
2. Add CoinSpot after CoinLoft, with a line break (`<br>`) at the end.

If you have any questions, please let me know.

Reply:
I have fixed this